src/components/slider/DoubleSlider.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from 'react';
+import styles from './doubleSlider.module.scss';
+import './slider.scss';
+import Slider from 'react-slider';
+import { useCategoryStore } from '../../store/useCategoryStore';
+
+type DoubleSliderProbs = {
+  title: string;
+  MIN: number;
+  MAX: number;
+  signs?: string;
+};
+
+export const DoubleSlider = ({ title, MIN, MAX }: DoubleSliderProbs) => {
+  const resetMin = useCategoryStore((state) => state.resetMin);
+  const resetMax = useCategoryStore((state) => state.resetMax);
+  const setResetMin = useCategoryStore((state) => state.setResetMin);
+  const setResetMax = useCategoryStore((state) => state.setResetMax);
+
+  const [values, setValues] = useState([MIN, MAX]);
```

Comment:
I have changed a way to save variables, unfornately I could not save MIN and MAX seperately because I had to pass setSliderPriceValues to the onChange method like that:

`{<Slider className={'slider'} onChange={setSliderPriceValues} value={sliderPriceValues} min={MIN} max={MAX} />}`

The setSliderPriceValues method gets a massive of two variables under the hood.

I use useCategoryStore to store vlues of price variables'

```
const setSliderPriceValues = useCategoryStore((state) => state.setSliderPriceValues);

  useEffect(() => {
    setSliderPriceValues([MIN, MAX]);
  }, []);
```